src/examples/xr_renderer.hpp

```diff
@@ -10,8 +10,9 @@ namespace jsar::example
   class XRStereoscopicRenderer
   {
   public:
-    XRStereoscopicRenderer(WindowContext *windowCtx)
+    XRStereoscopicRenderer(WindowContext *windowCtx, bool monoMode = false)
```

Comment:
This class will return a position between the left and right eye if it's mono mode.